source/get-started/virtual-machine-install/virt-manager.rst

```diff
@@ -155,51 +148,47 @@ This will launch the `New VM` dialog box, shown in figure 4 below.
+         proceed to the next step.
 
-#. In step 3 of the `New VM` wizard you set the memory and CPUs to allocate 
-   for your new VM. Depending on the resources available to you on your host
-   system, select what is appropriate. In this example I set `Memory` to
-   2048GB and `CPUs` to 1. Once complete, select the :guilabel:`Forward` as 
-   shown in figure 8, to go to the next step.
+#. Step 3 of the `New VM` wizard allocates the memory and CPUs for
+   the new VM. Choose settings that are valid for the resources on your host
+   system. This example sets `Memory` to 2048GB and `CPUs` to 1. Once complete,
+   select the :guilabel:`Forward` button as shown in Figure 8.
 
    .. figure:: figures/virtmgr/virt-manager-08.png
       :scale: 100%
       :alt: New VM Choose Memory and CPU settings dialog box
 
       Figure 8: New VM Choose Memory and CPU settings dialog box
 
-#. Step 4 of the `New VM` wizard will set up your storage media for your VM.
-   You can create a new disk image or select to use an existing image. For
-   this exercise we select the `Enable storage for this virtual machine` and
-   create a 20GB image to go with. Once set as shown in figure 9, select the
-   :guilabel:`Forward` button.
+#. Step 4 of the `New VM` wizard sets up the storage media for your VM. You
+   can create a new disk image or use an existing image. This example selects
+   `Enable storage for this virtual machine` and creates a 20GB image for it.
+   Once complete, select the :guilabel:`Forward` button as shown in Figure 9.
 
    .. figure:: figures/virtmgr/virt-manager-09.png
       :scale: 100%
       :alt: New VM Enable storage dialog box
 
       Figure 9: New VM Enable storage dialog box
 
-#. Step 5 of the wizard will show you the selections you have made and give
-   you the ability to customize the configuration before running the
-   installation. Select the `Customize configuration before install` check-box
-   and then select the :guilabel:`Finish` button.
+#. Step 5 of the `New VM` wizard displays the selections you made and allows
+   you to customize the configuration before running the installation. Select the
+   `Customize configuration before install` checkbox and select the
+   :guilabel:`Finish` button as shown in Figure 10.
 
    .. figure:: figures/virtmgr/virt-manager-10.png
       :scale: 100%
       :alt: New VM Ready to begin the installation dialog box
 
       Figure 10: New VM Ready to begin the installation dialog box
 
-#. Clear Linux requires an UEFI firmware to boot and operate correctly so you
-   need to customize the installation process by changing the firmware from
-   `BIOS` to `UEFI x86_64`. In the `Firmware` field, select the
-   `UEFI x86_64: /usr/share/qemu/OVMF.fd` entry as shown in figure 11 and 
-   select the :guilabel:`Apply` button.
+#. Customize the installation process by changing the firmware from `BIOS` to
+   `UEFI x86_64`. In the `Firmware` field, select the :file:`UEFI
```

Comment:
After `UEFI x86_64`, please add (portion of original):
"|CL| requires UEFI firmware."

UEFI firmware is almost universal with Linux distros, but it's worth noting here.